edx_exams/apps/lti/views.py

```diff
@@ -97,12 +96,39 @@ def start_proctoring(request, attempt_id):
     return redirect(get_lti_1p3_launch_start_url(launch_data))
 
 
-def end_assessment(request, attempt_id):  # pragma: no cover
+@api_view(['GET'])
+@require_http_methods(['GET'])
+@authentication_classes((JwtAuthentication,))
+@permission_classes((IsAuthenticated,))
+def end_assessment(request, attempt_id):
     """
     LTI End Assessment
     """
-    # TODO: Here we'd do all the end of assessment things.
-    attempt = ExamAttempt.objects.get(pk=attempt_id)
+    # Verify that the attempt associated with attempt_id exists.
+    attempt = get_attempt_by_id(attempt_id)
+    if not attempt:
+        return Response(
+            status=status.HTTP_400_BAD_REQUEST,
+            data={'detail': f'Attempt with attempt_id={attempt_id} does not exist.'}
+        )
+
```

Comment:
Note: I removed the check for an illegal status transition, because there is no status transition that's illegal when the `to_status` is `submitted`.